manila/share/drivers/netapp/dataontap/cluster_mode/lib_base.py

```diff
@@ -1973,6 +1973,14 @@ def revert_to_snapshot(self, context, snapshot, share_server=None):
                          self._get_backend_snapshot_name(snapshot['id']))
         LOG.debug('Restoring snapshot %s', snapshot_name)
         vserver_client.restore_snapshot(share_name, snapshot_name)
+        volume = vserver_client.get_volume(share_name)
+
+        # When calculating the size, round up to the next GB.
+        netapp_size = int(math.ceil(float(volume['size']) / units.Gi))
```

Comment:
This could be moved down to the if-condition, because it is not needed otherwise